modules/bazel-diff/7.2.5/MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(
+    name = "bazel-diff",
+    version = "7.2.5",
+    compatibility_level = 0,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
+bazel_dep(name = "rules_java", version = "7.5.0")
+bazel_dep(name = "rules_kotlin", version = "1.9.1")
+bazel_dep(name = "rules_jvm_external", version = "6.0")
+
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+maven.install(
```

Comment:
Since bazel-diff is a tool, it shouldn't share the default Maven dependency closure. Could you set a different name, e.g. `bazel_diff_maven`?